Source/WebKit/UIProcess/API/glib/WebKitWebView.cpp

```diff
@@ -5843,3 +5844,26 @@ webkit_web_view_get_default_content_security_policy(WebKitWebView* webView)
 
     return webView->priv->defaultContentSecurityPolicy.data();
 }
+
+/**
+ * webkit_web_view_end_immersive_session:
```

Comment:
I find a bit weird to have api to end an immersive session when there's no way if there's any immersive session or not, nor a way to start one. I guess the session is started by web content right? In that case I agree with @aperezdc that we should add a property to check if there's a session.